src/Propel/Runtime/Formatter/ArrayFormatter.php

```diff
@@ -88,7 +88,7 @@ public function getCollectionClassName(): ?string
      *
      * @return array|null
      */
-    public function formatOne(?DataFetcherInterface $dataFetcher = null): ?array
+    public function formatOne(?DataFetcherInterface $dataFetcher = null)
```

Comment:
Why is this needed?

Reply:
We have two child classes that return transfers.
`\Spryker\Zed\Kernel\Persistence\Repository\TransferObjectFormatter::formatOne()`
`\Spryker\Zed\Synchronization\Persistence\Propel\Formatter\SynchronizationDataTransferObjectFormatter::formatOne()`

Reply:
If there is no other way, please fix the docblock to

    @return array|mixed|null